WorkAndFun/src/main/java/pl/infoshare/workandfun/announcements/AnnouncementService.java

```diff
@@ -48,36 +52,44 @@ public AnnouncementService(AnnouncementsRepository announcementsRepository, AddA
     }
 
     public Announcement findById(Long id) {
+        LOGGER.debug("Repository request to find by id: {}", id);
         return announcementsRepository.findById(id).orElseThrow(() -> new AnnouncementNotFoundException(id));
     }
 
     public AddAndEditAnnouncementDto findByIdConvertToDto(Long id) {
+        LOGGER.debug("Repository request to find by id and convert to dto, id: {}", id);
         return addAndEditMapper.toDto(announcementsRepository.findById(id).orElseThrow(() -> new AnnouncementNotFoundException(id)));
     }
 
     public void deleteById(Long id) {
+        LOGGER.debug("Repository request to delete by id: {}", id);
         announcementsRepository.findById(id)
                 .ifPresentOrElse(announcementsRepository::delete, () -> {
                     throw new AnnouncementNotFoundException(id);
                 });
+        LOGGER.debug("Deleted announcement, id: {}", id);
     }
 
     public void save(AddAndEditAnnouncementDto dto) {
         Announcement announcement = addAndEditMapper.toEntity(dto);
         announcementsRepository.save(announcement);
+        LOGGER.info("Announcement successfully saved to database (id: {})", announcement.getId());
     }
 
     public Announcement update(Long id, AddAndEditAnnouncementDto dto) {
         Announcement entity = findById(id);
         BeanUtils.copyProperties(dto, entity);
+        LOGGER.info("Announcement successfully updated to database (id: {})", entity.getId());
```

Comment:
```suggestion
        LOGGER.info("Announcement successfully mapped into entity (id: {})", entity.getId());
        Announcement saved = announcementsRepository.save(entity);
        LOGGER.info("Announcement successfully updated in database (id: {})", entity.getId());
```
ja bym to rozdzielił: osobny logger że zmapowano Dto na encję i osobny na to ze z sukcesem zapisano do bazy danych.

Reply:
Loggery z mapowaniem są w mapperach na poziomie trace i chyba jest to błąd, który odkryłem dzięki tej uwadze. W aktualnej wersji przy odpalaniu strony głównej do pliku z logami idą 3 linijki logów mapowania na 1 ogłoszenie, co przy tysiącach ogłoszeń i odpalaniu strony głównej przez użytkowników tworzyłoby naprawdę grube logi.
Dzięki.